controllers/message-controller.ts

```diff
@@ -1,10 +1,8 @@
 /**
- * @file Controller RESTful Web service API for follows resource
+ * @file MessageController RESTful Web service API for messages resource
```

Comment:
Great job! Everything looks nice to me. Could you please just clarify why you deleted the message controller interface?